modules/core/core-content/src/main/java/com/enonic/xp/core/impl/content/ContentAuditLogSupportImpl.java

```diff
@@ -303,6 +304,15 @@ private void doPublish( final PushContentParams params, final PublishContentResu
             paramsSet.addStrings( "excludeChildrenIds", params.getExcludeChildrenIds().stream().
                 map( ContentId::toString ).collect( Collectors.toList() ) );
         }
+
+        if ( params.getExcludeWorkflowStates() != null )
```

Comment:
This makes me think that it is possible to publish invalid or in-progress content
Don't we need a guard for this is content service anyway?